<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPasswordGenerator</class>
 <widget class="QDialog" name="DialogPasswordGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>336</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Password generator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="3">
    <widget class="QCheckBox" name="checkSpecial">
     <property name="text">
      <string>Special (!, $, %, &amp;&amp;, /, \)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password generator</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="checkUpperCase">
     <property name="text">
      <string>Upper-case (A, B, C, ...)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="checkMinusAndPlus">
     <property name="text">
      <string>Minus, Plus (-, +)</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QCheckBox" name="checkBrackets">
     <property name="text">
      <string>Brackets ([, }, {, }, (, ), &lt;, &gt;)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QCheckBox" name="checkUnterlind">
     <property name="text">
      <string>Underline (_)</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QCheckBox" name="checkGenerateOnMouseMove">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Generate when mouse moving</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QCheckBox" name="checkSpaces">
     <property name="text">
      <string>Spaces ( )</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="checkDigits">
     <property name="text">
      <string>Digits (0, 1, 2, ...)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QPushButton" name="buttonGenerate">
     <property name="text">
      <string>Generat</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="checkLowerCase">
     <property name="text">
      <string>Lower-case (a, b, c, ...)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QSpinBox" name="textLengthOfPassword">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="value">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLineEdit" name="textGeneratedPassword"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="textMinNrOfUpperCase">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="specialValueText">
      <string>n.a.</string>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="displayIntegerBase">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="textMinNrOfLowerCase">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="specialValueText">
      <string>n.a.</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="textMinNrOfDigits">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="specialValueText">
      <string>n.a.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPasswordGenerator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPasswordGenerator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
